### DISCUSSION TITLE
Using rotki in a script.

### DISCUSSION THREAD

**Discussion by yusuf8ahmed** [rotki/rotki#5705] [Q&A] [answered]
Hello, 

I want to add rotki to a script that I am building but I don't want to use the internal database or MessagesAggregator as it is just a script.

Is this possible and if it is how can I implement it?

Current implementation is below:

```python

from rotkehlchen.exchanges.coinbase import Coinbase
from rotkehlchen.user_messages import MessagesAggregator

cb = Coinbase("cb-1", "api-key", b"api-secret", "database", MessagesAggregator)

print(cb.query_balances())
```

**Comment by kelsos** [comment, marked as answer]
Hey @yusuf8ahmed 

I have a small script that uses rotki for account-related operations for testing.
Maybe this can help you with your own script

https://github.com/kelsos/rotki-data-tool

**Comment by LefterisJP** [comment]
Also @yusuf8ahmed feel free to pop by our discord (have DMs open -- there is a captcha challenge): https://discord.gg/aGCxHG7 to ask more specific rotki development script quetions.

The tool by Kelsos is a bit more complex, but you can just initialize the classes that you need. The database is needed everywhere, but you can just make it it be in memory with a bit of hacking around
